### DISCUSSION TITLE
textual_inversion.py: filename changed

### DISCUSSION THREAD

**Discussion by Woisek** [vladmandic/sdnext#586] [Ideas]
Now that I can successfully train TI with Automatic again, I've noticed, that the filename for the loss has changed.
It's now called train.csv, but it should be **textual_inversion_loss.csv**
I have scripts that rely on this name to work. It would be highly appreciate if this could be renamed back to the original file. 🙂
Thanks!

**Comment by vladmandic** [comment]
fixed - i've made it configurable in settings. default will stay as train.csv, but you can set it to anything.

**Reply by Woisek** [reply]
Very cool! Thanks for this quick fix! 👍
